tests/unit/array.ts

```diff
@@ -10,14 +10,17 @@ function assertFrom(arrayable: any, expected: any[]) {
 	assert.deepEqual(expected, actual);
 }
 
-class MyArray {
-	constructor() {
-		Array.apply(this, arguments);
-		return this;
-	}
-	static from = array.from;
-}
-MyArray.prototype = Object.create(Array.prototype);
+// This breaks in the ES6 version for chrome,
```

Comment:
This chunk and it's associated test attempts to overwrite read-only prototype, breaks Chrome